flocker/node/test/test_deploy.py

```diff
@@ -1038,7 +1041,7 @@ def test_discover_manifestation_with_size(self):
         )
         d = api.discover_state(self.EMPTY_NODESTATE)
 
-        self.assertItemsEqual(
```

Comment:
The changes in this file look unrelated to the purpose of the patch. Given the scope of the patch, I'd strongly recommend making this change separately.